pkg/resources/kafka/kafka.go

```diff
@@ -213,7 +224,12 @@ func (r *Reconciler) Reconcile(log logr.Logger) error {
 			}
 			err = r.Client.Delete(context.TODO(), &corev1.ConfigMap{ObjectMeta: templates.ObjectMeta(fmt.Sprintf(brokerConfigTemplate+"-%s", r.KafkaCluster.Name, broker.Labels["brokerId"]), labelsForKafka(r.KafkaCluster.Name), r.KafkaCluster)})
 			if err != nil {
-				return errors.WrapIfWithDetails(err, "could not delete configmap for broker", "id", broker.Labels["brokerId"])
+				if apierrors.IsNotFound(err) {
```

Comment:
the same may apply to other broker pod dependent objects? `Service`, `Volumes/PVCs`

Reply:
Yes I am in the middle of the review and that is what I thought that if CM affected then the service and the PVC as well I guess.